test/e2e/common_suite_test.go

```diff
@@ -257,24 +266,47 @@ func (tc *testCase) run() {
 					}
 					t.Logf("Log output of peer pod:%s", LogString)
 				}
-
-				if tc.isAuth {
-					if err := getAuthenticatedImageStatus(ctx, client, tc.AuthImageStatus, *tc.pod); err != nil {
```

Comment:
Did you mean to delete the authenticated image tests? If not, can you please restore this and rebase this branch